src/styles/isomer-template.scss

```diff
@@ -1,5 +1,10 @@
 @charset "UTF-8";
 
+// This import is needed as we want the raw, un-scoped CSS to render the preview
```

Comment:
Hmmm why are we not scoping it to just be in the editormodule.scss ah?

Reply:
Good point! Didn't realise that we could scope it down from there, done in 5f8e7d6c53bee3a8881aae522f4505a89738fca2.